lib/credly/connection.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'faraday'
+
+module Credly
+  class Connection
+    def initialize(url, access_token)
+      @connection = Faraday.new(url: url) do |builder|
+        builder.request :json
+        builder.headers[:accept] = 'application/json'
+        builder.response :json
+        builder.request :authorization, 'Basic', -> { Base64.strict_encode64(access_token) }
```

Comment:
Do you need to use the `Base64.strict_encode64` method here? I think `access_token` should be passed just raw? Maybe I'm wrong, maybe there's something about Credly I don't know??

Reply:
As it turns out, yes, we should have just been using the method specified here: https://lostisland.github.io/faraday/#/middleware/included/authentication?id=basic-authentication
The method we were using does work, but we will switch it for the sake of clarity.